contracts/v2/PolygonRollupManager.sol

```diff
@@ -1238,6 +1302,32 @@ contract PolygonRollupManager is
         if (l1InfoRoot == bytes32(0)) {
             revert L1InfoTreeLeafCountInvalid();
         }
+
+        // In case of a chain in migration to PP, the inputs are a special case.
+        if (isRollupMigratingToPP[rollupID]) {
+            bytes32 expectedNewLocalExitRoot = rollup.lastLocalExitRoot;
+
+            // If the lastLocalExitRoot is zero, it means that the rollup has never verified a batch with bridges
+            if (rollup.lastLocalExitRoot == bytes32(0)) {
+                // To proof the transition, the expected newLocalExitRoot must be to root of an empty 32 levels tree
+                expectedNewLocalExitRoot = _EMPTY_TREE_ROOT;
+            }
+
```

Comment:
reuse the `expectedNewLocalExitRoot`  to not check again storage slot ( in the if)